base/applications/taskmgr/perfdata.c

```diff
@@ -389,6 +390,11 @@ void PerfDataRefresh(void)
                     pPerfData[Idx].GDIObjectCount = GetGuiResources(hProcess, GR_GDIOBJECTS);
                 }
 
+                if (IsWow64Process(hProcess, &bIsWow64) && bIsWow64)
```

Comment:
Could this move inside `if (pSPI->UniqueProcessId != (HANDLE)0x4)`?

Reply:
I suppose it could, as it's unlikely that that the system process is a WOW64 process, therefore I don't think it matters much. The comment above the if statement describes why it's there in the first place. Since that explanation does not apply to this code, I've decided to put the it outside of that scope.